### ISSUE TITLE
Warning "Unexpected unnamed function  func-names" in new ESLint rule

### ISSUE THREAD

**Issue by RespectMyAuthoritah** [btholt/complete-intro-to-react#83]
my eslint version is 4.4.1, it would give a warning like this:
![func express](http://i.imgur.com/cxUO7sf.png)

when defining functions in such a way:
`const funct = function () {
    ....
}`

http://eslint.org/docs/rules/func-names

**Comment by khamiltonuk** [comment]
I don't think this is a problem with  ESLint You might want to change your function to 
`const funct = () => { .... }`
A bit late but hopefully it helps someone else

**Comment by tewang1** [comment]
i come across this question too

**Comment by wa05** [comment]
but how to bind this when you have to use functions like requiredIf of Vuelidate for example..

**Comment by daniloas80** [comment]
You can also switch off this eslint warning. You just need to write  **_'func-names': 'off'_** inside rules definition on your **_.eslintrc.js_** script.

**Comment by breno-felipe** [comment]
@ daniloas80 Following your recommendation to use r 'func-names': 'off' in eslintrc.js solved the problem I was having in my code
